apps/docs/content/docs/registry.mdx

```diff
@@ -0,0 +1,218 @@
+---
+title: Registry
+description: Run your own code registry.
+---
+
+import { Step, Steps } from 'fumadocs-ui/components/steps';
+import Image from 'next/image';
+
+You can use the `shadcn` CLI to run your own code registry. Running your own registry allows you to distribute your custom components, hooks, pages, config, rules and other files to any project.
+
+<Callout type="warn">
+  **Note:** If you want to build universal components we recommend building your registry with React Native, even thought the registry works with any project type and any framework, and is
+  not limited to React.
+</Callout>
```

Comment:
I'm not sure I understand this note. I think the registry docs in rnr should stick to what applied to rnr and link to the shadcn docs for additional resources like you did at the bottom.